### ISSUE TITLE
500 post N9 setup

### ISSUE THREAD

**Issue by sp222222** [GerryDazoo/Slinger#221]

**Comment by sp222222** [comment]
Gerry, have you seen this?
https://www.reddit.com/r/slingbox/comments/xk9c12/remote_control_setup_can_it_be_changed_after_119/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by sp222222** [comment]
M box works after 15min delay but not the 500. anyway to put our own server string in that to fool it?

**Comment by GerryDazoo** [comment]
Yes. But...
Did you notice that the custom remote files are only 1Kb. Not big enough to house all the data needed for the remote IR codes and JPEG for the remote picture. So somehow the slingbox is getting all this info off the web based on the info in the custom remote file. So when he says he blocked internet access I don't believe it. Maybe he meant just the sling servers.
Anyway, the thread from the guy who say's he want's to build this own replacement has some legs. We have packet captures of the whole process and now the code that sling runs to do it. I'd like to try to convince him to take it on first instead of messing with the streaming part.

**Comment by jkaponis** [comment]
I think that the skin for he remote control is not uploaded to the slingbox, i say that because yeas ago  i create a custom remote and the tutorial i follow say that the skin will be the default skin of the slingbox that means that the skin is not downloadable from the box and is loaded in the slingplayer.

https://slingbox.fandom.com/wiki/How_to_create_custom_remote_upgrade

**Comment by sp222222** [comment]
Yes he needs to do that. time is of the essence. I might ping him about
that part ….I’m off work Monday if you need anything.


Yeah this bin files are too small and it sounded too good to be true.

**Comment by GerryDazoo** [comment]
Yes. The skin is not stored on the slingbox.
